unfoldingWord/translationCore
# Fixed Non-working build on Mac

#### This pull request addresses:

- Fixed Non-working build on Mac 


#### How to test this pull request:

- Run `npm run release-osx`
- After the mac build is done then open `translationCore/release/macos-x64/translationCore-macos-x64-0.8.1-alpha.14.dmg` to install the build
- The app should open with no problems


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3692)
